services/QuillLMS/app/services/demo/create_admin_report.rb

```diff
@@ -4,24 +4,36 @@ class Demo::CreateAdminReport
 
   NUMBER_OF_CLASSROOMS_TO_DESTROY_SESSIONS_FOR = 20
   RANGE_OF_NUMBER_OF_SESSIONS_TO_DESTROY = 14..28 # 10-20% of 140
+  BATCH_DELAY = 1.minute
 
   attr_reader :teacher_email
```

Comment:
I realize this isn't new code, but what about initializing data and getting rid of the @passed_data instance variable.

```
attr_reader :data, :delay, :teacher_email

def initialize(teacher_email, passed_data=nil, delay=BATCH_DELAY)
    @teacher_email = teacher_email
    @data = passed_data || Demo::SessionData.new.admin_demo_data
    @delay = delay
  end
```

This gets rid of the def data method that has memoization and disjunction which can be difficult to parse.